### ISSUE TITLE
context.publish throws an error

### ISSUE THREAD

**Issue by melalj** [tjmehta/coworkers-test#9]
I'm using coworkers-test to test a that a queue publish properly a message:

``` js
self.publish('exchange-name', 'routing.key', [{ message: 'test' }], {});
```

And I keep getting `TypeError: Cannot read property 'publish' of undefined`
The undefined here is not `self`, but `publisherChannel` on https://github.com/tjmehta/coworkers/blob/master/lib/context.js#L121

It looks like the method `context.publish` is not stubbed properly (maybe it's using a spy instead of a stub in the mockFactory)

Thanks ;)

**Comment by tjmehta** [comment]
Out of town today, will check it out tonight. I can pull a PR on the go if you have a fix

Thanks!

**Comment by tjmehta** [comment]
fixed in coworkers-test@0.3.1
